src/plugins/data_view_editor/public/plugin.tsx

```diff
@@ -63,6 +67,25 @@ export class DataViewEditorPlugin
       userPermissions: {
         editDataView: () => dataViews.getCanSaveSync(),
       },
+      /**
+       * Helper method to generate a new data view editor service.
+       * @returns DataViewEditorService
+       */
+      dataViewEditorServiceFactory: ({
```

Comment:
What functionality is being used from the service? Perhaps less functionality should be exposed.

Reply:
Nevermind! The service is a bit simpler than I remember. That doesn't happen very often! 😄